ztag/transforms/https.py

```diff
@@ -15,7 +15,10 @@ def __init__(self, *args, **kwargs):
         super(HTTPSTransform, self).__init__(*args, **kwargs)
 
     def _transform_object(self, obj):
-        tls = obj['data']['tls']
+        try:
+            tls = obj['data']['tls']
+        except (KeyError, TypeError):
```

Comment:
Actually, I believe TypeError would indicate that obj or obj['data'] was malformed by zgrab2 or ztag code, going to change this back to just handling KeyError.